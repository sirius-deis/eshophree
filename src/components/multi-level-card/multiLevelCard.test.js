import { render, screen } from "@testing-library/react";
import MultiLevelCard from "./multiLevelCard";


describe("MultiLevelCard component", () => {
  it("should match snapshot", () => {
    const { container } = render(<MultiLevelCard />);
    expect(container).toMatchSnapshot();
  });
  it("should render content inside component", () => {
    render(<MultiLevelCard>
      <div>Content</div>
    </MultiLevelCard>);
    expect(screen.getByText("Content")).toBeInTheDocument();
  });
});