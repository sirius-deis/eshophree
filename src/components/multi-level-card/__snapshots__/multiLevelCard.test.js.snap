// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiLevelCard component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB dbXCYH"
  />
</div>
`;
